### ISSUE TITLE
[concert_conductor_graph] status of busy client

### ISSUE THREAD

**Issue by jihoonl** [robotics-in-concert/rocon_qt_gui#134]
concert conductor graph does not expose any information about _busy_ client.

It would be helpful if it can tell what is its remote_controller and what's running inside

**Comment by stonier** [comment]
Yeah, it needs a few things, but most of these are pending on #129
